flymake-eslint.el

```diff
@@ -38,15 +38,17 @@
   :group 'programming
   :prefix "flymake-eslint-")
 
-(defcustom flymake-eslint-executable-name "eslint"
-  "Name of executable to run when checker is called.
-Must be present in variable `exec-path'."
-  :type 'string
-  :group 'flymake-eslint)
-
-(defcustom flymake-eslint-executable-args nil
```

Comment:
completely removing this symbol will cause this package to break for any user who has assigned a value to it.

I suggest continuing to support it for now but apply the `define-obsolete-variable-alias` macro to tell users they should prefer `flymake-eslint-executable` instead.

Reply:
That is true. I welcome you to edit the code as you see fit to respect the older symbols.